app/src/main/java/com/x8bit/bitwarden/data/vault/repository/VaultRepositoryImpl.kt

```diff
@@ -1355,6 +1307,78 @@ class VaultRepositoryImpl(
             .onSuccess { vaultDiskSource.saveFolder(userId, it) }
     }
     //endregion Push Notification helpers
+
+    @Suppress("LongMethod")
+    private suspend fun syncInternal(userId: String): SyncFido2CredentialsResult {
```

Comment:
Can we make this return type more generic, this call is not tied directly to Fido2